### PR TITLE
Upgrade dependencies for webpack and styles

### PR DESCRIPTION
**Pull request by dolfolife**
⚠️ 
TODO:
- [x] Fix styles.
- [x] Fix location of dist

I think I still need to work on this to load styles properly. It seems like the error on https://github.com/buttercomponents/butter-component-settings that does not load settings in on this.

### CODE REVIEW

File: webpack.build.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 var path = require('path');
 var webpack = require('webpack');
-var ExtractTextPlugin = require('extract-text-webpack-plugin');
```
Comment (xaiki) [inline webpack.build.js:3]: that maybe why styles are failing ?

**Comment by xaiki** [comment]
@btzr-io you're the style master, want to help @rodolfo2488 to know why styles aren't loading with this ?

**Comment by btzr-io** [comment]
@xaiki I don't really know much about webpack / react,  and can't  build `xaiki-react` 😞
It's there an error or something in the console?

File: webpack.build.js (hunk 1 above)
Comment (dolfolife) [reply webpack.build.js:3]: it was never used

**Comment by xaiki** [comment]
this is still for `butter-component-settings`

**Comment by btzr-io** [comment]
xaiki: what you mean? `butter-component-settings` is working fine ^^
well can't build it:  `error: TypeError('Path must be a string. Received ' + inspect(path))`
Sent a PR https://github.com/buttercomponents/butter-component-builder/pull/4 ^^

**Comment by btzr-io** [comment]
Anyway I'm still don't know what's the problem...
how can I test dist/`bundle.js`?

**Comment by xaiki** [comment]
@btzr-io run `npm run build` and copy `dist/bundle.js` over to butter-desktop

**Comment by btzr-io** [comment]
@xaiki ~here? -> `butter-desktop/build` ( but don't open settings )~
```JS
Uncaught Error: Cannot find module 'butter-component-settings'
```
Never mind -> `butter-desktop/build/*/node_modules/butter-component-settings` ^^

**Comment by btzr-io** [comment]
```JS
Uncaught Error: addComponentAsRefTo(...): Only a ReactOwner can have refs. 
```
@xaiki got an error ( still can't open settings ) ^^

**Comment by xaiki** [comment]
@btzr-io yeah that's the issue we're tracking...

**Comment by btzr-io** [comment]
@xaiki what? then how do you know that styles aren't loading if you can't load the component / react?
https://gist.github.com/jimfb/4faa6cbfb1ef476bd105

**Comment by xaiki** [comment]
@btzr-io with this branch the styles aren't loading in `butter-component-settings` it complains about postcss in many directories

**Comment by btzr-io** [comment]
@xaiki  I don't see nothing about `postcss`
not sure where / how are you testing that?

**Comment by btzr-io** [comment]
@rodolfo2488 @xaiki  why do we need `postcss-loader`?

**Comment by btzr-io** [comment]
Merged with https://github.com/buttercomponents/butter-component-builder/pull/5

**Pull request merged**